src/js/components/DateInput/__tests__/DateInput-test.js

```diff
@@ -274,10 +274,13 @@ describe('DateInput', () => {
     );
     expect(container.firstChild).toMatchSnapshot();
 
+    // new calendar logic adjust start date by default
     fireEvent.click(getByText('10'));
     expect(onChange).toHaveBeenCalled();
     expect(onChange).toHaveReturnedWith([
-      '2020-07-02T00:00:00-08:00',
+      // Calendar is returning UTC, but should return local
+      // this should be resolved when https://github.com/grommet/grommet/issues/4352 ågets fixed
```

Comment:
Can we fix this as part of this change?

Reply:
It could be if I'm able to sort out that timezone issue soon. I didn't want to hold up this PR waiting for that to merge.